src/uct/ib/rc/accel/rc_mlx5_iface.c

```diff
@@ -707,11 +707,11 @@ UCS_CLASS_INIT_FUNC(uct_rc_mlx5_iface_common_t, uct_iface_ops_t *tl_ops,
     self->tm.cmd_wq.super.super.type = UCT_IB_MLX5_OBJ_TYPE_LAST;
     init_attr->rx_hdr_len            = UCT_RC_MLX5_MP_ENABLED(self) ?
                                        0 : sizeof(uct_rc_mlx5_hdr_t);
+    init_attr->max_rd_atomic         = IBV_DEV_ATTR(dev, max_qp_rd_atom);
```

Comment:
i see this is a constructor of uct_rc_mlx5_iface_common_t, isn't it also used from dc_mlx5 ?

Reply:
good catch, need to move to `uct_rc_mlx5_iface_t` constructor